challenges.js

```diff
@@ -11,14 +11,15 @@ const challengeSort = {
 	*/
 	async getApiToArray () {
 		try {
```

Comment:
Maybe use a try, catch, finally blocks with a bool and if-statetment inside finally to know if it should return the empty array or not.

Reply:
Otherwise good initiative.

Reply:
Okey i will do that, thanks : )